packages/hardhat-kadena/src/plugin.ts

```diff
@@ -209,6 +213,10 @@ extendConfig((config, userConfig) => {
             customChains: etherscanCustomChains,
             enabled: true,
           };
+          config.blockscout = {
```

Comment:
This should be documented in the README. One of the example projects should be updated to use this config. The other one should keep the etherscan config. That will show how they both work.